services/status_service.py

```diff
@@ -15,6 +15,9 @@ def players_status(self):
         if not game:
             raise NotCreatedGame()
 
+        if game.game_status == "finished":
+            return game.get_status()
+
         if game.game_status != "started":
             raise NotStartedGame()
```

Comment:
This lines could be simplified in one validation like:
`if game.game_status == "created":
    raise NotStartedGame()
`